documentation/restore.md

```diff
@@ -0,0 +1,44 @@
+# Restore Puppet Enterprise using the PEADM module
+
+Once you have a [backup](backup.md) you can restore a PE primary server. It is important to highlight that you can not use the `peadm::restore` plan with a backup that was not created with the `peadm::backup` plan.
+
+**Things to consider**
+
+There is some downtime involved when the `peadm:restore` plan is executed. The following services will be restarted:
+
+1. pe-console-services
+2. pe-nginx
+3. pxp-agent
+4. pe-puppetserver
+5. pe-orchestration-services
+6. puppet
+6. pe-puppetdb
+
+There is also a procedure related to the restoration of the databases where `peadm::restore` will temporarily set privileges (permissions) to a DB user. These temporary privileges are removed right after the PostgreSQL restore command finishes.
```

Comment:
What do you mean by 'set privileges to a DB user'?

Reply:
I added a bit more of details about it.

>There is also a procedure related to the restoration of the databases where `peadm::restore` will temporarily set privileges (permissions) to a DB user with the only purpose of restoring the database. These temporary privileges are removed right after the PostgreSQL restore command finishes.